guidelines/enhancement_template.md

```diff
@@ -272,6 +272,40 @@ to implement the design.  For instance,
  > 1. This requires exposing previously private resources which contain sensitive
   information.  Can we do this?
 
+## Security
+
+This section should cover the security impact of the change, if any. For example,
+does it introduce new privileges or capabilities? Does it expose new risks?
+
+### Credentials and RBAC
+
+What credentials will the new component be using if it needs to talk to either the control plane or other components? What RBAC needs to be applied?
```

Comment:
Also possibly bringing up under the security umbrella is that if there are credentials or certificates or other auth involved, how will those get rotated in the event of a leak, or just out of best-practice rotation processes.  And then also with notes about how long they are expected to last while a cluster is hibernated, and if/how they can be bootstrapped after a cluster that has hibernated for too long is re-awakened?